tests/interlocked_win32_ut/mock_interlocked.c

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "windows.h"
+
+#undef InterlockedAdd 
```

Comment:
symbols should be undef'd only if they are #defined previously. Otherwise grizzly compiler will complain about it.

Interesting this did not happen here... are these all macros?

Reply:
The Interlocked functions defined in windows.h are macros, so I have to undef before redefining them to be the mocked versions.

---
In reply to: [427739418](https://github.com/Azure/azure-c-util/pull/43#discussion_r427739418) [](ancestors = 427739418)